virtualmachine/vsphere/vm.go

```diff
@@ -516,9 +516,9 @@ type Flavor struct {
 	// constants [FlavorLarge, FlavorSmall, FlavorMedium, FlavorCustom]
 	Name string
 	// Represents the number of CPUs
-	NumCPUs int32
+	NumCPUs int32 `json:"cpu"`
 	// Represents the size of main memory in MB
-	MemoryMB int64
+	MemoryMB int64 `json:"memory"`
```

Comment:
should we rename the variable name also ?

Reply:
Here NumCPUs and MemoryMB says correctly what parameter represents, so I think keeping the name as it is has no harm and will be better for code readability.